README.md

```diff
@@ -21,18 +21,28 @@ remotes::install_github("pharmaverse/admiral.test", ref = "devel")
 # How To Update
 
 Firstly, make a GitHub issue in this repo with the planned updates and tag `@pharmaverse/admiral` so that one of the development core team can sanity check the request.
-Then there are two main ways to extend the test data, either by adding new datasets or extending existing datasets with new records/variables.
+Then there are two main ways to extend the test data, either by adding new datasets or extending existing datasets with new records/variables. Whichever method you choose, it is worth noting the following:
+
+-   Programs that generate test data are stored in the `dev/` folder.
+-   Each of these programs is written as a standalone R script: if any packages need to be loaded for a given program, then call `library()` at the start of the program.
+-   Most of the packages that you are likely to need will already be specified in the `renv.lock` file, so they will already been installed if you have been keeping in sync--you can check this by entering `renv::status()` in the Console. However, you may also wish to install `{metatools}` and `{ggplot2}`, which are currently not specified in the `renv.lock` file. If you feel that you need to install any other packages in addition to those just mentioned, then please tag `@pharmaverse/admiral` to discuss with the development core team.
+-   When you have created a program in the `dev/` folder, you need to run it as a standalone R script, in order to generate a test dataset that will become part of the {admiral.test} package, but you do not need to build the package.
+-   Each dataset is stored as a `.rda` file whose name is consistent with the name of the dataset: for example, the dataset `dm` should be renamed to `raw_dm` before saving it as `raw_dm.rda`; if you save `dm` as `raw_dm.rda` and subsequently load the `.rda` file, then `dm` (not `raw_dm`) will be loaded into the global environment.
+-   The programs in `dev/` are stored within the {admiral.test} GitHub repository, but they are **not** part of the {admiral.test} package--the `dev/` folder is specified in `.Rbuildignore`.
+-   When you run a program that is in the `dev/` folder, you generate a dataset that is written to the `data/` folder, which will become part of the {admiral.test} package.
```

Comment:
Perhaps include a link to the [exported data chapter](https://r-pkgs.org/data.html#sec-data-data) in the r-packages book use as a guide/best practices and if they need further details?

Reply:
@zdz2101 , thanks for your suggestion. I've now added a link to the "Exported data" chapter.